lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -4926,13 +4926,16 @@ DefMacroI('\medbreak',   undef, '\par');
 DefMacroI('\bigbreak',   undef, '\par');
 
 DefMacroI('\line', undef, '\hbox to \hsize');
-DefConstructor('\leftline{}', sub {
+DefMacro('\leftline Undigested',   '\ltx@leftline{\hbox{#1}}');
+DefMacro('\rightline Undigested',  '\ltx@rightline{\hbox{#1}}');
+DefMacro('\centerline Undigested', '\ltx@centerline{\hbox{#1}}');
+DefConstructor('\ltx@leftline{}', sub {
     alignLine($_[0], $_[1], 'left'); },
   bounded => 1);
-DefConstructor('\rightline{}', sub {
+DefConstructor('\ltx@rightline{}', sub {
```

Comment:
You're going to want to add `alias=>'\rightline'` (and similar) to this kind of construct.